src/mixins/_mixins.scss

```diff
@@ -327,3 +327,9 @@
   -ms-overflow-style: none; // hide scrollbar but keep element scrollable for IE/Edge
   scrollbar-width: none; // hide scrollbar on firefox
 }
+
+@mixin fd-typography($fd-size) {
+  font-family: var(--sapFontHeaderFamily);
+  color: var(--sapGroup_TitleTextColor);
```

Comment:
if you want this mixin to be more reusable you can pass font-color as a parameter and set the default value to `var(--sapGroup_TitleTextColor)`. If you don't have to change it you can omit it